CrewChiefV4/iRacing/Sim.cs

```diff
@@ -37,52 +34,35 @@ public Sim()
         private SessionData _sessionData;
         public SessionData SessionData { get { return _sessionData; } }
 
+        private int _DriverId;
+        public int DriverId { get { return _DriverId; } }
+
         private Driver _driver;
         public Driver Driver { get { return _driver; } }
 
         private Driver _paceCar;
         public Driver PaceCar { get { return _paceCar; } }
-        #endregion
 
-        #region Methods
-        
-        public void Reset()
-        {
-            _mustUpdateSessionData = true;
-            _mustReloadDrivers = true;
-            _currentSessionNumber = null;
-            _driver = null;
-            _paceCar = null;
-            _drivers.Clear();
-            _telemetry = null;
-            _sessionInfo = null;
-        }
-
-        #region Drivers
 
         private readonly List<Driver> _drivers;
         public List<Driver> Drivers { get { return _drivers; } }
 
-        private bool _isUpdatingDrivers;
-
-        private void UpdateDriverList(SessionInfo info)
+        private void UpdateDriverList(SessionInfo info, bool reloadDrivers)
         {
-            this.GetDrivers(info);
+            this.GetDrivers(info, reloadDrivers);
             this.GetResults(info);
         }
 
-        private void GetDrivers(SessionInfo info)
+        private void GetDrivers(SessionInfo info, bool reloadDrivers)
         {
-            if (_mustReloadDrivers)
+            if (reloadDrivers)
             {
-                Console.WriteLine("MustReloadDrivers: true");
+                Console.WriteLine("Reloading Drivers");
                 _drivers.Clear();
                 _driver = null;
-                _paceCar = null;
-                _mustReloadDrivers = false;
             }
 
-            // Assume max 70 drivers
+            // Assume max 128 drivers
```

Comment:
this comment suggests the for loop should go up to 128, but it doesn't - am i missing something?

Reply:
ups me testing